public/js/lib/colorContraster.js

```diff
@@ -31,12 +31,24 @@ function parseColor(color) {
 
 function parseHex(color) {
   // #ff00de
-  // support for 3 character hex strings would be nice :)
-  // pull request me if you like - noopkat
-  const justColor = color.substring(1, 7);
-  const r = parseInt(justColor.substring(0, 2), 16); // hexToR
-  const g = parseInt(justColor.substring(2, 4), 16); // hexToG
-  const b = parseInt(justColor.substring(4, 6), 16); // hexToB
+  let r; let g; let b; let justColor;
+  
+  // need to check here if a 3 digit short hex code is provided
+  // We're assuming here that the color will always be 
+  // (simple string length test would do)
+  if (color.length === 4) {
+    justColor = color.substring(1, 4);
+    // Using [String.prototype.repeat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat)
```

Comment:
I don't think we need this comment since we're supporting es6+ browsers now.

Reply:
@underscoredotspace I believe you're right. Let me remove the comment as well